packages/ui/src/hooks/use-preview.ts

```diff
@@ -32,6 +34,9 @@ const usePreview = (context: Context): [boolean, PreviewConfig?] => {
     // Init i18n
     void initI18n();
 
+    // Block pointer event
+    document.body.classList.add(conditionalString(styles.preview));
```

Comment:
nit: can we use a react node and provider-context to control the preview mask?

Reply:
like, add an additional layer on top?

Reply:
sry, you are adding just a class, my mistake, I thought you were adding a `div`